src/MiniHallway.jl

```diff
@@ -0,0 +1,88 @@
+# Mini Hallway problem defined in http://cs.brown.edu/research/ai/pomdp/examples/mini-hall2.POMDP.
+# Original idea published in Littman, Cassandra and Kaelbling's ML-95 paper.
+
+# Basic parameters are:
+# discount: 0.950000
+# values: reward
+# states: 13
+# actions: 3
+# observations: 9
+# The rest is available at link
+
+
+struct Observation
```

Comment:
What is the reason for introducing a new type here. Wouldn't Int work?

Reply:
This is a relic from the first versions where I kept multiple parameters which I found out were unnecessary. 
I felt that the problem definition would be more readable with the wrapper structure.
I agree that it is also overkill, so I changed that.